lib/begin.go

```diff
@@ -181,6 +193,15 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if urlerr, ok := err.(*url.Error); ok {
+			if operr, ok := urlerr.Err.(*net.OpError); ok {
+				if dnserr, ok := operr.Err.(*net.DNSError); ok {
+					if dnserr.Err == "no such host" {
```

Comment:
It even still boils down to a string comparison. I was really hoping for a `net.ErrNoSuchHost`.